packages/components/buttons/ToggleButton.tsx

```diff
@@ -17,30 +15,34 @@ interface ToggleButtonProps extends SwitchProps {
 export default function ToggleButton({
   isActive = false,
   onValueChange,
+  disabled,
   ...rest
 }: ToggleButtonProps) {
-  const [focused, setFocused] = useState(isActive ?? false);
-
   function onValueChangeHandler(value: boolean) {
-    setFocused(value);
-
     if (!onValueChange) {
       return;
     }
 
     onValueChange(value);
   }
 
+  const getIOSBackgroundColor = () => {
+    if (disabled) {
+      return neutral99;
+    }
+
+    return isActive ? blueDefault : neutral33;
+  };
+
   return (
     <Switch
-      // @ts-expect-error: the active thumb color is a weird green that can't be changed without "activeThumbColor"
-      activeThumbColor={secondaryColor}
```

Comment:
this change breaks the thumb color, like written in the comment
<img width="170" alt="Screenshot 2024-02-25 at 10 09 51" src="https://github.com/TERITORI/teritori-dapp/assets/7917064/e67c7aa4-d8e2-43dd-9456-53dce4590b53">

Reply:
Thanks, I didn't realize the issue was on web.